Source/Core/DolphinQt/Config/Mapping/MappingCommon.cpp

```diff
@@ -63,16 +66,23 @@ QString DetectExpression(QPushButton* button, ciface::Core::DeviceContainer& dev
   QApplication::processEvents();
 
   // Avoid that the button press itself is registered as an event
-  Common::SleepCurrentThread(100);
+  Common::SleepCurrentThread(50);
```

Comment:
Is this one even needed with the event filter? It felt odd even in the old code now that I look closely...

Reply:
It does help when the button actually gets pressed with the "space" or "return" key.